influence_benchmark/RL/conversation_collator.py

```diff
@@ -122,5 +147,15 @@ def torch_call(self, examples: List[Union[List[int], Any, Dict[str, Any]]]) -> D
                 assistant_response_start_idxs[num_messages_to_ignore + 1 :],
             ):
                 batch["labels"][i, user_template_start:assistant_response_start] = self.ignore_index
+
+            # Mask tool calls and tool responses
+            for tool_call_start in tool_call_start_idxs:
+                next_start = next((x for x in start_header_idxs if x > tool_call_start), len(batch["labels"][i]))
+                batch["labels"][i, tool_call_start:next_start] = self.ignore_index
+
+            for tool_response_start in tool_response_start_idxs:
+                next_start = next((x for x in start_header_idxs if x > tool_response_start), len(batch["labels"][i]))
+                batch["labels"][i, tool_response_start:next_start] = self.ignore_index
```

Comment:
Did not double check this with the debugger. Potential source of bugs if we see issues with this in the future